content/homeserver/pricing.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Matrix.org Homeserver Pricing"
++++
+
+One of the missions of the Matrix.org Foundation is to nurture and grow the ecosystem and the network. To support this mission, the Foundation provides the public Matrix.org homeserver as an easy entry point for those wanting to create their first Matrix account and join the network. The Matrix.org homeserver has played a key role in the growth of the ecosystem since the project started in 2014.
+
+To help pay for the costs associated with providing a public homeserver, the Foundation is introducing premium accounts as announced in the blog post [here](/blog/2025/06/funding-homeserver-premium).
+
+These new usage plans are being introduced in a phased manner and so not all new users will be placed on a plan automatically.
+
+Where a user is not currently on a plan they may be changed to a free plan in the future, and will be notified in advance.
+
+## Your Data, Your Choice
+
+Matrix is a decentralised network by design, and the goal is to give the users the ability to choose where they would like to create their accounts, as it then gives them control over where their data is stored and who is looking after them.
+
+This means that you don’t have to be using the Matrix.org homeserver. You could choose to use another shared homeserver or you could have your own dedicated homeserver.
+
+For those that want to have their own homeserver the Foundation maintains a list of [known hosting providers](/ecosystem/hosting/) that might be suitable for your needs.
+
+## Pricing
+
+Details of pricing have not yet been finalised, but will be published here once usage plans will be made available.
+
+Pricing may change over time, as described in the [Homeserver Terms](/legal/terms-and-conditions).
+
+## Usage limits
+
+Where a user account has been allocated to a plan the following usage limits apply:
+
+|Usage|Free plan|Premium plan|
```

Comment:
This table is pretty much unreadable using the default formatting. I'm not sure what the correct approach to fix is? (should I convert to an html table?)

Reply:
I meant the rendering of the table, not the markdown formatting.

Here is how it looks:

![image](https://github.com/user-attachments/assets/356549b7-801c-4846-86b0-a1db8f009c17)

Which I think is unreadable.